File: src/main/java/com/harleylizard/magic_things/common/MagicThingsConfiguredFeatures.java
package com.harleylizard.magic_things.common;

import net.minecraft.core.registries.Registries;
import net.minecraft.resources.ResourceKey;
import net.minecraft.world.level.levelgen.feature.ConfiguredFeature;

public final class MagicThingsConfiguredFeatures {
    public static final ResourceKey<ConfiguredFeature<?, ?>> FOULED_TREE = ResourceKey.create(Registries.CONFIGURED_FEATURE, MagicThings.resourceLocation("fouled_tree"));

}

File: src/main/kotlin/com/harleylizard/magic_things/common/MagicThingsConfiguredFeatures.kt
package com.harleylizard.magic_things.common

import com.harleylizard.magic_things.common.MagicThings.Companion.resourceLocation
import net.minecraft.core.registries.Registries
import net.minecraft.resources.ResourceKey
import net.minecraft.world.level.levelgen.feature.ConfiguredFeature

object MagicThingsConfiguredFeatures {
    val fouledTree: ResourceKey<ConfiguredFeature<*, *>> = ResourceKey.create(Registries.CONFIGURED_FEATURE, "fouled_tree".resourceLocation)

}